public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -72,6 +71,11 @@ const getStyles = (theme: GrafanaTheme2, showContextButton: boolean, isInDashboa
       visibility: hidden;
       width: ${showContextButton ? '80px' : '40px'};
     `,
+    logRowMenuCell: css`
+      position: absolute;
+      right: ${isInDashboard ? '40px' : '163px'};
```

Comment:
Unfortunately I can not really justify those numbers. That's just what I figured out.

Reply:
I am slightly worried if this is not going to break if users have different default font sizes/different browser. You see how I have it in the ~middle of log line. 

What about having menu at the beginning of the line instead of end? This way it would be at the same position for all cases - wrap/not wrapped. 

<img width="1390" alt="image" src="https://user-images.githubusercontent.com/30407135/193241574-9d1246a1-5479-43df-9b64-7210164570ce.png">

<img width="647" alt="image" src="https://user-images.githubusercontent.com/30407135/193241522-78f80cfe-56a1-41ab-8bfd-9690c3cc0217.png">

Reply:
I just noticed another issue with this, where the timestamp row got a wrong width. I moved this issue back to draft to figure out a better solution for this.

Reply:
Oh, regarding the issue you are seeing, that is indeed only if there's just the "one" button menu. I think the positions are correct if you add a line filter to also see the show context button.

Reply:
> I am slightly worried if this is not going to break if users have different default font sizes/different browser

That's a good argument for using `em` or `rem` instead of `px`. https://blog.logrocket.com/using-em-vs-rem-css/#what-em-rem-why-use

Reply:
I think "custom" font-sizes are currently not supported in Grafana? I changed the font-size in Chrome and it did not apply to Grafana.